pom.xml

```diff
@@ -85,6 +85,11 @@
   </properties>
 
   <dependencies>
+    <dependency>
+      <groupId>com.google.guava</groupId>
```

Comment:
Why can't we add it only in `e2e-tests` profile?

Reply:
>To address these circumstances, Maven supports build profiles. Profiles are specified using a subset of the elements available in the POM itself (plus one extra section), and are triggered in any of a variety of ways

I think the current POM doesn't have a direct guava dependency and adding the overriding dependency in `profile` won't work then.

Reply:
Can we add the dependency with test scope?

Reply:
Done